lib/offsite_payments/integrations/poli_pay.rb

```diff
@@ -0,0 +1,269 @@
+module OffsitePayments
+  module Integrations
+    module PoliPay
+      def self.notification(post, options = {})
+        Notification.new(post, options)
+      end
+
+      def self.return(query_string, options = {})
+        Return.new(query_string, options)
+      end
+
+      def self.sign(fields, key)
+        OpenSSL::HMAC.hexdigest(OpenSSL::Digest::SHA256.new, key, fields.sort.join)
+      end
+
+      class Interface
+        include ActiveUtils::PostsData # ssl_get/post
+
+        def self.base_url
+          "https://poliapi.apac.paywithpoli.com/api"
+        end
+
+        def initialize(login, password)
+          @login = login
+          @password = password
+        end
+
+        private
+
+        def standard_headers
+          authorization = Base64.encode64("#{@login}:#{@password}")
+          {
+            'Content-Type' => 'application/json',
+            'Authorization' => "Basic #{authorization}"
+          }
+        end
+
+        def parse_response(raw_response)
+          JSON.parse(raw_response)
+        end
+
+        class RequestError < StandardError
+          attr_reader :exception, :message, :error_message, :error_code
+
+          def initialize(exception)
+            @exception = exception
+
+            @response      = JSON.parse(exception.response.body)
+            @success       = @response['Success']
+            @message       = @response['Message']
+            @error_message = @response['ErrorMessage']
+            @error_code    = @response['ErrorCode']
+          end
+
+          def success?
+            !!@success
+          end
+
+          def errors
+            ERRORS
```

Comment:
Where is this defined @mnoack ? Can't see a definition for this class. Is the method even used?